test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can we rename the file to `.eslintrc.json` as without extension is deprecated?
https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Reply:
Uh I didn't know this one. Updated 👍

Reply:
Sorry we actually have two `.eslintrc`s. Can you update `test/.eslintrc` as well? 🙏

Reply:
Oops sorry my bad. Fixed that too 👍